exporter/prometheusremotewriteexporter/README.md

```diff
@@ -45,7 +48,7 @@ Example:
 ```yaml
 exporters:
   prometheusremotewrite:
-    endpoint: "http://some.url:9411/api/prom/push"
+    endpoint: "https://my-cortex:7900/api/v1/push"
```

Comment:
@tigrannajaryan @bogdandrutu The example here is failing the link check. How do you normally deal with example links?

Reply:
The linker checker has a (not very nice) way to disable checking a particular link: https://github.com/gaurav-nelson/github-action-markdown-link-check#disable-check-for-some-links

I wish there was a better way (maybe there is?).

Reply:
Thanks for the pointers! Reverted this change back not to deal with it.